src/DotNetty.Handlers/Tls/TlsHandler.Writer.cs

```diff
@@ -173,7 +178,18 @@ private void Wrap(IChannelHandlerContext context)
                         _lastContextWritePromise = promise;
                         if (buf.IsReadable())
                         {
+#if NETCOREAPP_2_0_GREATER || NETSTANDARD_2_0_GREATER
+                            var asyncWrite = WriteAsync(buf, promise);
+                            if (!asyncWrite.IsCompleted)
+                            {
+                                var asyncWriteTask = asyncWrite.AsTask();
+                                asyncWriteTask.Ignore();
```

Comment:
how does this `Ignore()` work? Doesn't Task still throw not even considering the continuations? Ignore code is the following:
```
private static readonly Action<Task> IgnoreTaskContinuation = t => { _ = t.Exception; };

...
_ = task.ContinueWith(
                    IgnoreTaskContinuation,
                    CancellationToken.None,
                    TaskContinuationOptions.OnlyOnFaulted | TaskContinuationOptions.ExecuteSynchronously,
                    TaskScheduler.Default);
```

Reply:
I made a repro - indeed it ignores the exception. It's me who does not know the async code :(

```
using System;
using System.Threading.Tasks;

var a = Task.Run(async () =>
{
    Console.WriteLine("task run");
    await Task.Delay(1000);
    throw new Exception("qwe");
});

var b = a.ContinueWith((task => _ = task.Exception));

// await a; // throws
await b; // ignores!

Console.WriteLine("finish");
```